.github/workflows/phpunit.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         php-version: ${{ matrix.php }}
         tools: composer:v1
         extensions: bcmath, curl, dom, mbstring, pcntl, sockets, xml
-        coverage: none
+        coverage: xdebug
```

Comment:
It will slow down all preparation and tests, yes?

Reply:
Maybe move this to matrix -> if this value === xdebug -> add  xdebug  and  run coverage ?